### DISCUSSION TITLE
Iamforce2 not starting

### DISCUSSION THREAD

**Discussion by DawidWenderski** [TheKikGen/MPCLiveXplore-libs#14] [Q&A]
Hello, first of all, great work! Thank You.

Second - I establish SSH connection on MPC LIVE 2 with flashed card, and bootstramp 2.11.8 (When I clicked update from USB, I got communicate about downgrade from version 2.12.2, but I proceed).

MPC Reboot -> KiKGen logo ->

1. I was trying to SSH comman via WINSCP but I failed
2. Then I try with Windows CMM, and same with power shield (picture bellow) ->MPC Reboot after second command, but IAMFORCE2  not started.  

What can I do? :-(

 
![!!!!!!FORCE](https://github.com/TheKikGen/MPCLiveXplore-libs/assets/93607744/41e643ab-075b-492a-8f41-1f5af2f471f3)

**Comment by TheKikGen** [comment]
1/ Download the last 2.12.2 SSH Img here : https://github.com/TheKikGen/MPC-LiveXplore
Check the link at the top.   You must enter the "ssh img" sub directory, then download the mpc-2.12.2.tkglssh-update.img file.

2/ Copy that file on an sdcard or the root of an external hardisk, then boot your MPC, and choose update from USB.  Apply the update : you will come back to 2.12.2 but this time with a full SSH access, and capability to start a bootstrat. Nothing else was modified.

3/ Follow the steps to setup IamForce2 (read documentation [here](https://github.com/TheKikGen/MPCLiveXplore-libs/wiki)), and update Iamforce2 from the github as mentioned [here ](https://github.com/TheKikGen/MPCLiveXplore-libs/wiki#updating-an-existing-bootstrap-installation)

**Comment by TheKikGen** [comment]
Do not update the Force image obviously.

**Reply by DawidWenderski** [reply]
I have done fresh flash with bootstramp and do as instruction says (without another update from sd card), and now i can see FORCE logo after boot for 10 minutes right now, and nothing more. I will stay it like this for some minnutes, and see what will happend.

**Reply by DawidWenderski** [reply]
Unfortunately, FORCE does not want to start fully. I'm currently at the stage where after a reboot the MPC+KIKGEN logo appears, then the FORCE logo appears and the KIKGEN flashes on it for a while, then the FORCE sign alone remains and nothing else loads.

I deleted the update file manually, because with the command I could not (a message about the absence of such a file appeared). Update file through the command also does not work, a message about the absence of such a location appears.

![!!!!!!FORCE4](https://github.com/TheKikGen/MPCLiveXplore-libs/assets/93607744/f267b0a6-0a6c-4414-8f04-daf879e46689)

**To sum up** - I did an update to 2.12. I then inserted the card with bootstramp (without doing a re-upgrade(downgrade) from the SD card) and after adding the appropriate force start command to the dooer file and rebooting, the only thing that starts up is the FORCE wallpaper. 

Below I paste what my dooer file looks like:

![!!!!!!FORCE3](https://github.com/TheKikGen/MPCLiveXplore-libs/assets/93607744/82b0e9c1-9c22-4647-acd9-32d20797ff9d)


I would appreciate your help.

**Reply by DawidWenderski** [reply]
UPDATE 

I manualy update force module to 3.3 and edit file to launch force from this file. Still the same - Force logo shows on boot with kitgen blinking, and nothing happens :/

**Reply by DawidWenderski** [reply]
Meaby the problem is micro sd card that I use in adapter?

**Reply by DawidWenderski** [reply]
It seems that I have checked all possibilities, in the last 3 days, I even understood how it all works xD

Unfortunately, trying with three versions of root force, I got no result other than the endless FORCE logo.

Without your help I will not move on.

**Reply by DawidWenderski** [reply]
Solved by Lord KikGen by himself ^^ For some peoples there with update problem - remember to connect MPC into web ^^ 

This Force on MPC is amazing!!!
